openapi.yaml

```diff
@@ -474,6 +474,167 @@ paths:
          responses:
             '200':
                description: Success
+      get:
+         summary: Get a single device
+         description: Get a single device managed by an organization given device id.
```

Comment:
What do you think about something like: "Get a device by id (serial number)" for the description?

It seems helpful to immediately communicate that the id is the serial!

Reply:
That sounds great! Do we want we also include "managed by an organization", since both this endpoint and the get-devices endpoint requires orgId, and the get-devices description does include "managed by an organization"

Reply:
I'm wondering about that as well. Since almost all of our endpoints will require an orgId for the foreseeable future (the only exception being GET List Organizations), I think it is okay for us to establish a pattern of omitting it from the brief summary— it can be taken as a given. Otherwise, for consistency we might want to add it to all the descriptions and that seems like overkill. I'm good to remove it from get-devices for the same reason. 

We could also add more information to the intro page https://docs.managexr.com/api-reference/introduction to highlight that most endpoints will require an orgId and are for devices within an org. Wdyt @quan-managexr ?

Reply:
I think we can remove it from the "get device" endpoint while keeping it in the "get devices" endpoint. 

This is because:
1. The summary doesn't have to be strictly consistent between the endpoints.
2. What should be consistent about the summary across endpoints is the communication of the use case and most important params.

In the case of the "get device" endpoint, the most important param is deviceId (so we include that in the summary), while for the "get devices" endpoint, it's orgId (so we should include orgId in the summary).

Reply:
Sounds good to me! We'll adjust the description of the get-devices endpoint anyways with the query changes 👍